projects/dotcms-field-elements/src/components/dot-key-value/dot-key-value.tsx

```diff
@@ -35,15 +33,30 @@ import {
 export class DotKeyValueComponent {
     @Element() el: HTMLElement;
 
+    /** (optional) Placeholder for the key input text in the <key-value-form> */
+    @Prop() formKeyPlaceholder: string;
+
+    /** (optional) Placeholder for the value input text in the <key-value-form> */
+    @Prop() formValuePlaceholder: string;
+
+    /** (optional) The string to use in the key label in the <key-value-form> */
+    @Prop() formKeyLabel: string;
+
+    /** (optional) The string to use in the value label in the <key-value-form> */
+    @Prop() formValueLabel: string;
+
+    /** (optional) Label for the add button in the <key-value-form> */
+    @Prop() formAddButtonLabel: string;
+
+    /** (optional) The string to use in the delete button of a key/value item */
+    @Prop() listDeleteLabel: string;
+
     /** (optional) Disables field's interaction */
     @Prop() disabled = false;
```

Comment:
don't we need the reflectToAttr?

Reply:
I didn't add them in the host `props` because I want to investigate a little bit the effects of that.